android/TriblerApp/app/src/main/AndroidManifest.xml

```diff
@@ -100,7 +101,7 @@
         </activity>
         <activity
             android:name=".BeamActivity"
-            android:label="@string/title_activity_beam"
+            android:launchMode="singleTop"
```

Comment:
You are looking at the BeamActivity, not the MainActivity, but I indeed only had set it on SearchActivity before.